hpedockerplugin/volume_manager.py

```diff
@@ -451,6 +458,12 @@ def _create_snapshot(self, src_vol_name, schedName, snapshot_name,
             LOG.debug(msg)
             response = json.dumps({u"Err": msg})
             return response
+        snap_cpg = None
+        if 'snap_cpg' in vol and vol['snap_cpg']:
+            snap_cpg = vol['snap_cpg']
+        else:
+            snap_cpg = vol.get('cpg', self._hpeplugin_driver.get_cpg
```

Comment:
Here don't fetch the cpg field for snapcpg
as in 3.0 volume will have both cpg and snap_cpg please update snap_cpg with snap_cpg field

also change the driver cpg fetch definition